src/output/influx.rs

```diff
@@ -0,0 +1,176 @@
+use std::sync::{Arc, Mutex};
+use std::time::Duration;
+
+use futures::future::Future;
+use tokio;
+
+use influent::create_client;
+use influent::client::{Client, Credentials, ClientError, Precision};
+use influent::measurement::{Measurement, Value};
+
+use item::Item;
+
+use output::AKOutput;
+use output::error::*;
+
+#[derive(Clone, Deserialize)]
+pub struct InfluxOutput {
+    #[serde(default = "influx_database_default")]
+    pub database: String,
+    pub username: String,
+    pub password: String,
+    #[serde(default = "influx_hosts_default")]
+    pub hosts: Vec<String>,
+    #[serde(default = "influx_raw_as_fallback_default")]
+    pub use_raw_as_fallback: bool,
+    #[serde(default = "influx_always_raw_default")]
+    pub always_write_raw: bool,
+    #[serde(skip, default = "influx_dummy_client_deser")]
+    pub client: Option<Arc<Mutex<Client + Send>>>,
+}
+
+fn influx_database_default() -> String {
+    String::from("antikoerper")
+}
+
+fn influx_hosts_default() -> Vec<String> {
+    vec![String::from("http://localhost:8086")]
+}
+
+fn influx_raw_as_fallback_default() -> bool {
+    false
+}
+
+fn influx_always_raw_default() -> bool {
+    false
+}
+
+fn influx_dummy_client_deser() -> Option<Arc<Mutex<Client + Send>>> {
+    None
+}
+
+impl std::fmt::Debug for InfluxOutput {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        write!(f, "InfluxOutput {{ database: {}, username: {}, password: {}, hosts: {:?}, use_raw_as_fallback: {}, always_write_raw: {} }}",
```

Comment:
Did you know that you can do `write!(f, "some = {thing}", thing = self.foobar)`? That would make that macro call a bit less complicated, because you'd have a name for each value you pass to the format.

Just a suggestion, tho.